docs/user/alerting/alert-types.asciidoc

```diff
@@ -1,159 +1,28 @@
 [role="xpack"]
 [[alert-types]]
-== Standard stack alert types
+== Alert types
```

Comment:
What do you think about removing the word "type" from the headings.  I don't feel that it adds any value.  So the navigation would look like this:

+ Actions and connectors
+ Alerts
    Index threshold
    ES query

Reply:
I think it should say something about the different 'Types of alerts'.. Maybe keeping this as it is should be fine.